### PR TITLE
feat(CI): Set up export-service with postgres container

### PR DESCRIPTION
**Pull request by ahmed-nour-fdc**
* Create and run postgres container in integrations tests and docker-compose.
* Run export-service in integration tests.
* Removed the duplicate docker compose files, and the default one now is earthly.
* `kuberpult` make target now maps to the target `kuberpult-earthly`.

### CODE REVIEW

File: database/migrations/postgres/1720455590920137_resize_environments_json.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE environments ALTER COLUMN json TYPE varchar(1000000);
```
Comment (AminSlk) [inline database/migrations/postgres/1720455590920137_resize_environments_json.up.sql:1]: Why should we put a limit on the number of characters? we can use `VARCHAR` without any limits, like similar columns in other tables.
```suggestion
ALTER TABLE environments ALTER COLUMN json TYPE VARCHAR;
```

File: database/migrations/sqlite/1718277940006100_environments.up.sql [hunk 2]
```diff
@@ -3,6 +3,6 @@ CREATE TABLE IF NOT EXISTS environments
     created TIMESTAMP,
     version BIGINT,
     name VARCHAR(255),
-    json VARCHAR(255),
+    json VARCHAR(1000000),
```
Comment (AminSlk) [inline database/migrations/sqlite/1718277940006100_environments.up.sql:6]: Same here

File: charts/kuberpult/setup-postgres.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    echo "logs:"
+    kubectl -n "$ns" logs -l "$label" || echo "could not get logs for $label"
+    echo "describe pod:"
+    kubectl -n "$ns" describe pod -l "$label"
+    echo ...
+  done
+}
+
+earthly ../../infrastructure/docker/postgres+docker
+kind load docker-image postgres:local
```
Comment (sven-urbanski-freiheit-com) [inline charts/kuberpult/setup-postgres.sh:22]: don't we need a `docker pull` here too?

File: database/migrations/postgres/1718277940006100_environments.up.sql [hunk 4]
```diff
@@ -1 +0,0 @@
-../sqlite/1718277940006100_environments.up.sql
```
Comment (sven-urbanski-freiheit-com) [inline database/migrations/postgres/1718277940006100_environments.up.sql:1]: why was this deleted?

File: charts/kuberpult/setup-postgres.sh (hunk 3 above)
Comment (ahmed-nour-fdc) [reply charts/kuberpult/setup-postgres.sh:22]: The image is generated from the above earthly command `earthly ../../infrastructure/docker/postgres+docker`

File: database/migrations/postgres/1720428454051313_fk_deployments.up.sql [hunk 5]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE deployments
+    ADD COLUMN transformerEslId INTEGER
```
Comment (sven-urbanski-freiheit-com) [inline database/migrations/postgres/1720428454051313_fk_deployments.up.sql:2]: there should not be any changes in existing migrations. Please create new migrations instead.

**Review by sven-urbanski-freiheit-com — CHANGES_REQUESTED** [review]
.

File: database/migrations/postgres/1718277940006100_environments.up.sql (hunk 4 above)
Comment (ahmed-nour-fdc) [reply database/migrations/postgres/1718277940006100_environments.up.sql:1]: This was a symbolic link to the environments migrations file for sqlite. To solve the json size limit problem, we agreed to create new migrations file for postgres, but edit the migrations file for sqlite directly. Since this file is a symbolic link, it also changes by changing the original file.
The solution to this was to remove the symbolic link, and create a file with the same name and content.

File: tests/integration-tests/cluster-setup/setup-postgres.sh [hunk 6]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
```
[the remark this answers is not in the document]
Comment (ahmed-nour-fdc) [reply tests/integration-tests/cluster-setup/setup-postgres.sh:5]: Will be taken care of in a follow-up story.
